### ISSUE TITLE
Notifications and search button not appearing

### ISSUE THREAD

**Issue by dajix350** [frabcus/instalamb#1]
Hi! Thank you so much for making this extension, it is so incredibly helpful. I use Instagram on my phone through a Firefox PWA, with uBlock Origin and this installed. 

I've noticed that even when hide search and hide notifications is disabled, the search button does not appear, and the notification menu is empty. The search button reappears and the notifications area regains function when I disable the extension.

![Screenshot_20240506-082754](https://github.com/frabcus/instalamb/assets/74509183/1821cc43-cc6a-46b9-82ac-ff9736d29f9d)
![Screenshot_20240506-082912](https://github.com/frabcus/instalamb/assets/74509183/752e376b-2c3d-468b-a6a2-db0bc325ac45)
![Screenshot_20240506-082916](https://github.com/frabcus/instalamb/assets/74509183/181065d5-ada9-47ef-82bc-f943e67b6e3c)

**Comment by frabcus** [comment]
Thanks so much for your bug report!

Took me a little while to work it out as it's slightly confusing. On mobile web the Instagram Explore navigation menu looks like an hourglass. So hiding "Explore" navigation also hides "Search" - they're the same page.

<img width="198" alt="image" src="https://github.com/user-attachments/assets/22314da9-9b4e-4db2-8706-ef76c8029cda">

Is that fine for what you do, or did you want to also hide the search content on the mobile explore page, so it just has the search text entry at the top?

I've updated the preferences to describe what Explore means better.
https://github.com/frabcus/instalamb/commit/391a5670f9aa9d0a08419a9aaa4c8f5d701af038

The empty notifications screen was a serious bug on mobile impacting lots of pages, which I think I've fixed here.
https://github.com/frabcus/instalamb/commit/f14892770c7ef4f3686340fca6219c9a118fd1ae

I'll make a new release soon!
